ampd/src/lib.rs

```diff
@@ -353,8 +373,15 @@ where
         let broadcaster = self.broadcaster.client();
         let sub = self.event_subscriber.subscribe();
 
+        let event_config = EventProcessorConfig {
+            retry_delay,
+            retry_max_attempts,
+            stream_timeout,
+            stream_buffer_size: 10000,
+        };
```

Comment:
why is the config defined here?

Reply:
Because we have a `consume_events` a few lines down that needs a event config as input. But `create_handler_task` receives the individual fields.